### PR TITLE
Ajaxrequests

### PR DESCRIPTION
**Pull request by carolinegreen**
Remove promises to make location picker engine compatible wiht IE 9, 10, and 11.  Also change sorting to both before and after uniqBy, as the new uniqBy without promises changes the order of things.  Update tests to reflect this.  Paired with @tvararu

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,7 @@
 ## [Unreleased]
 
 _(add items here for easier creation of next log entry)_
+Fix - Allow support for IE 9, 10 and 11.
```
Comment (tvararu) [inline CHANGELOG.md]: Should be a bullet item like:

```md
- Fix - …
```

File: __mocks__/fetch.js [hunk 2]
```diff
@@ -1,17 +1,20 @@
 const graph = require('../examples/location-picker-graph.json')
```
Comment (tvararu) [inline __mocks__/fetch.js]: We can remove this file, we don't need the mock anymore. :)

File: lib/uniq.js [hunk 3]
```diff
@@ -1,16 +1,22 @@
...
-
-    map.has(key) || map.set(key, item)
-
-    return map
-  }, new Map()).values()]
+// Inspired by https://stackoverflow.com/questions/840781/easiest-way-to-find-duplicate-values-in-a-javascript-array
+export default (inputArray, callbackToPresentableName) => {
+    var sorted_arr = inputArray.slice().sort((left, right) => {
+    	if (callbackToPresentableName(left) < callbackToPresentableName(right)) {
+    		return -1
```
Comment (tvararu) [inline lib/uniq.js]: This file is indented weirdly.

File: CHANGELOG.md (hunk 1 above)
Comment (carolinegreen) [inline CHANGELOG.md]: Yeah

File: __mocks__/fetch.js (hunk 2 above)
Comment (carolinegreen) [inline __mocks__/fetch.js]: Oh right! :)

File: lib/uniq.js (hunk 3 above)
Comment (carolinegreen) [inline lib/uniq.js]: :s

**Pull request merged**